docs/CODEOWNERS

```diff
@@ -1,8 +0,0 @@
-# See https://github.com/OSS-Docs-Tools/code-owner-self-merge
-
-README.ru.md @demensky @virzak [translation] [README-RU]
-README.cn.md @Kerminate @kunl [translation] [README-CN]
-README.ja.md @MaySoMusician @sasurau4 [translation] [README-JA]
-README.ko.md @Ailrun @cimfalab [translation] [README-KO]
-README.es.md @LuigiBaute @kant [translation] [README-ES]
-README.pt.md @thzoid @cdias900  [translation] [README-PT]
```

Comment:
Can we leave this in a form that github will still auto-add these people to PRs that touch these files? I didn't know we had any dedicated people lined up to review them.

Reply:
I don't think so, you can't assign someone to a PR if don't have write access